packages/cli/test/dev/utils.js

```diff
@@ -260,8 +275,11 @@ async function testFixture(directory, opts = {}, args = []) {
   });
 
   dev._kill = dev.kill;
-  dev.kill = async (...args) => {
-    dev._kill(...args);
+  dev.kill = async () => {
+    // kill the entire process tree for the child as some tests
+    // spawn child processes that we don't have the pid for
+    await nukeProcessTree(dev.pid);
```

Comment:
I don't think this is necessary anymore, since `stop()` should be taking care of teardown when the main dev process is killed.

Reply:
Turns out this `nukeProcessTree()` is still needed. For whatever reason when we run the `invalid-env-var-name` integration test on Linux and it spawns `vc dev` with `{ shell: true }`, killing the process leaves the shell process in a "defunct" state. I guess "defunct" processes are a Linux thing and happen when the process is corrupt, killed, or in this case has a child process `start.js` that hasn't exited.